applications/crossbar/doc/api_authentication.md

```diff
@@ -12,9 +12,9 @@ Get your API key for your account:
 * It can also be accessed by system administrators directly from the database by using `curl` to request the account doc from CouchDB:
 
 ```shell
-curl -v -X PUT \
+curl -v -X GET \
     -H "X-Auth-Token: {AUTH_TOKEN}" \
-    http://localhost:15984//accounts/{ACCOUNT_ID}
+    http://localhost:15984/accounts/{ACCOUNT_ID}/api_key
```

Comment:
@k-anderson this needs reverting.

Get api_key from Couch:
`curl http://localhost:15984/accounts/{ACCOUNT_ID} | grep pvt_api_key`

Get it from Kazoo:
`curl -H 'X-Auth-Token: {AUTH_TOKEN}' http://localhost:8000/v2/accounts/{ACCOUNT_ID}/api_key`

as documented in
https://github.com/2600hz/kazoo/blob/master/applications/crossbar/doc/accounts.md#fetch-the-accounts-api-key
https://github.com/2600hz/kazoo/blob/master/applications/crossbar/doc/api_authentication.md#getting-your-api-key-from-the-api-must-already-authenticate-as-a-user